examples/consuming-services/consuming_services.md

```diff
@@ -1,4 +1,4 @@
-# Consuming services: client objects
```

Comment:
Shouldn't file names also change for consistency?

Reply:
Are we doing that? Usually, we follow the simple case for file names.

Reply:
I meant just consum**e**_services instead of consum**ing**_services.

Reply:
Ahh, yes, we need to do that.

Reply:
It is ok to keep the file names as otherwise, it will change the URLs and create broken links.